lib/smart_answer_flows/calculate-statutory-sick-pay.rb

```diff
@@ -302,48 +302,36 @@ def define
       end
 
       # Answer 1
-      outcome :already_getting_maternity
+      outcome :already_getting_maternity, use_outcome_templates: true
```

Comment:
Can these individual `use_outcome_templates: true`s be replaced with a global `use_outcome_templates` for this flow?